lock-api/build.gradle

```diff
@@ -37,6 +38,7 @@ dependencies {
     implementation 'com.palantir.tokens:auth-tokens'
     implementation 'com.palantir.tritium:tritium-annotations'
     implementation 'io.dropwizard.metrics:metrics-core'
+    implementation 'org.jetbrains:annotations'
```

Comment:
Don't we have this annotation in guava or something?

Reply:
Yes changing